packages/donations-app/app/components/project-wizard/projectWizardOutcomeView.html

```diff
@@ -117,6 +117,26 @@
     </div>
   </div>
 
+  <div class="form-group">
+    <label class="col-sm-3 control-label">Action verb for progress</label>
+
+    <div class="col-sm-9">
+      <input type="text" class="form-control" ng-model="outcome.actionVerbForProgress"
+             placeholder="Outcome action verb for progress, example: help"
+             name="actionVerbForProgress" required>
+    </div>
+  </div>
+
+  <div class="form-group">
+    <label class="col-sm-3 control-label">Unit for action verb</label>
```

Comment:
Let's put a more verbose description with an example ("A part of text displayed on my impact splash beside "We raised $100 to [help]...", it's easy to forget what is the true meaning of the parameter.